client/src/game/events/keyboard.ts

```diff
@@ -68,6 +68,12 @@ export function onKeyDown(event: KeyboardEvent): void {
             event.preventDefault();
             event.stopPropagation();
             gameStore.toggleUI();
+        } else if (event.key === "r" && event.ctrlKey) {
```

Comment:
Ctrl+R is a common keybinding to reload a webpage and I would thus prefer not to override it, as it could be that a user wants to actually hard reload the page if something seems wrong.  I would either choose a different combo (be it a different modifier or letter) or even just use the R key without any modifier.

Reply:
You're right. But it's difficult to find one that's really unique and maybe  not in use by other apps. In the most ideal situation you'd let the user pick a combo. I wouldn't use a non-modified letter since that might interfere with typing actions during 'normal' operations.

Reply:
I understand the sentiment, but this is something that should already be properly handled.  The 'd' key deselects the current selection, but not if the user is typing in a textfield/textarea as can be seen in lines 21-23